bot/cogs/tags.py

```diff
@@ -17,6 +18,19 @@ class Tags:
     Save new tags and fetch existing tags.
     """
 
+    FAIL_TITLES = [
+        "Please don't do that.",
+        "You silly tart.",
```

Comment:
Probably remove this specific one.

Reply:
oh damn it, you silly tart >.>
<.<
jk c: